packages/cli/src/controller/migrate/constants.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import {NETWORK_FAMILY} from '@subql/common';
+import {convertEthereumDs, convertEthereumTemplate} from './manifest/ethereum';
+import {DsConvertFunction, TemplateConvertFunction, NetworkExampleProject} from './types';
+
+// @ts-ignore
+export const networkDsConverters: Record<NETWORK_FAMILY, DsConvertFunction> = {
+  [NETWORK_FAMILY.ethereum]: convertEthereumDs,
+  // [NETWORK_FAMILY.substrate]: convertSubstrateDs,
+  // [NETWORK_FAMILY.cosmos]: convertCosmosDs,
+  // [NETWORK_FAMILY.algorand]: convertAlgorandDs,
+  // [NETWORK_FAMILY.flare]: convertFlareDs,
+  // [NETWORK_FAMILY.near]: convertNearDs,
+  // [NETWORK_FAMILY.stellar]: convertStellarDs,
+  // [NETWORK_FAMILY.concordium]: convertConcordiumDs,
+};
+
+// @ts-ignore
+export const networkTemplateConverters: Record<NETWORK_FAMILY, TemplateConvertFunction> = {
+  [NETWORK_FAMILY.ethereum]: convertEthereumTemplate,
+  // [NETWORK_FAMILY.substrate]: convertSubstrateDs,
+  // [NETWORK_FAMILY.cosmos]: convertCosmosDs,
+  // [NETWORK_FAMILY.algorand]: convertAlgorandDs,
+  // [NETWORK_FAMILY.flare]: convertFlareDs,
+  // [NETWORK_FAMILY.near]: convertNearDs,
+  // [NETWORK_FAMILY.stellar]: convertStellarDs,
+  // [NETWORK_FAMILY.concordium]: convertConcordiumDs,
+};
+
+export const graphToSubqlNetworkFamily: Record<string, NETWORK_FAMILY> = {
+  'ethereum/contract': NETWORK_FAMILY.ethereum,
+  // Add more mappings here as needed
+};
+
+// TODO, alternative approach
+export const subqlNetworkTemplateNetwork: Record<NETWORK_FAMILY, NetworkExampleProject> = {
+  Algorand: {},
+  Concordium: {},
+  Cosmos: {},
+  Flare: {},
+  Near: {},
+  Stellar: {},
+  Substrate: {},
+  Ethereum: {
+    '1': {
+      name: 'eth-starter',
+      path: 'Ethereum/ethereum-starter',
+      description: '',
+      remote: 'https://github.com/subquery/ethereum-subql-starter',
+    },
+  },
+};
+
+// @ts-ignore
+export const graphNetworkNameChainId: Record<NETWORK_FAMILY, Record<string, string>> = {
+  [NETWORK_FAMILY.ethereum]: {mainnet: '1'},
```

Comment:
Mainnet just means any ethereum network i thought. Do we need to prompt the user?

Reply:
No it refer to a specific network https://thegraph.com/docs/en/developing/supported-networks/
example from poap: 
https://github.com/poap-xyz/poap-subgraph/blob/aa640881ffd6ca5f50930ad32b78006f51e4a626/subgraph.template.yaml#L9
https://github.com/poap-xyz/poap-subgraph/blob/aa640881ffd6ca5f50930ad32b78006f51e4a626/config/goerli.json#L2

Reply:
But good we found https://thegraph.com/docs/_next/data/IlewALDCf3LFk3rL28g-h/en/developing/supported-networks.json